packages/package_info/lib/package_info.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+/// Provides information about the current application package.
+class PackageInfo {
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

We should probably just make these top-level functions and leave it up to callers to import with a namespace if they choose.